templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,12 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 6.0
 
+**Feature level 155**
+
+* [`GET /messages`](/api/get-messages): The new `include_anchor`
+  parameter controls whether a message with ID matching the specified
+  `anchor` should be included.
```

Comment:
Do we want to add a mention that the original `update_message_flags` endpoint will no longer include in the event any messages that already had the desired flags state?